src/ripple/proto/org/xrpl/rpc/v1/ledger.proto

```diff
@@ -55,10 +55,29 @@ message RawLedgerObject
     }
 
     ModificationType mod_type = 3;
+
+    bytes predecessor = 4;
```

Comment:
Would you add comments to these protobuf definitions, similar to `get_ledger.proto`?